File: code/1006/1006.c
#include <stdio.h>

int main() {

  float num1;
  float num2;
  float num3;

  int num1_weight = 2;
  int num2_weight = 3;
  int num3_weight = 5;

  float average;

  scanf("%f %f %f", &num1, &num2, &num3);

  average = (num1 * num1_weight + num2 * num2_weight + num3 * num3_weight) /
            (num1_weight + num2_weight + num3_weight);

  printf("MEDIA = %.1f\n", average);

  return 0;
}

File: code/1006/1006.rb
input_num1 = gets
input_num2 = gets
input_num3 = gets

num1 = input_num1.to_f
num2 = input_num2.to_f
num3 = input_num3.to_f

num1_weight = 2
num2_weight = 3
num3_weight = 5

average = (num1 * num1_weight + num2 * num2_weight + num3 * num3_weight) / (num1_weight + num2_weight + num3_weight)

puts "MEDIA = #{'%.1f' % average.round(1)}"
